chainerx/_fallback_workarounds.py

```diff
@@ -62,6 +62,13 @@ def _from_chx(array, check_backprop=True):
     # Objects with other types are kept intact.
     # Returns a pair: (xp, cupy device or dummy context, numpy/cupy.ndarray).
     if not isinstance(array, chainerx.ndarray):
+        if (isinstance(array, numpy.ndarray) or
+                cupy and isinstance(array, cupy.ndarray)):
```

Comment:
A nitpick, but it better to bring `or` to the front of the next line.
(See [PEP-8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).)
Also, as this expression mixes `or` and `and`, it's better to put parentheses around the `and` expression.

Reply:
Changed! Thanks :D